test/cmd/apply.sh

```diff
@@ -310,8 +310,8 @@ __EOF__
   kubectl delete -f hack/testdata/multi-resource-1.yaml "${kube_flags[@]:?}"
 
   ## kubectl apply multiple resources with one failure during builder phase.
-  # Pre-Condition: No configmaps
-  kube::test::get_object_assert configmaps "{{range.items}}{{${id_field:?}}}:{{end}}" ''
+  # Pre-Condition: Only "kube-root-ca.crt" configmap exists.
+  kube::test::get_object_assert configmaps "{{range.items}}{{${id_field}}}:{{end}}" 'kube-root-ca.crt:'
```

Comment:
scope the precondition to the actual config map we're inspecting:

```
kube::test::get_object_assert 'configmaps --field-selector=metadata.name=foo' "{{range.items}}{{${id_field:?}}}:{{end}}" ''
```

Reply:
done